bindings-test/src/tests.rs

```diff
@@ -126,6 +126,12 @@ fn asset_info_not_found() {
 fn query_positions() {
     let mtps: Vec<Mtp> = vec![Mtp {
         address: "user".to_string(),
+        collateral_asset: "".to_string(),
+        trading_asset: "".to_string(),
+        liabilities_asset: "".to_string(),
+        custody_asset: "".to_string(),
+        take_profit_borrow_rate: Decimal::zero(),
+        open_price: Decimal::zero(),
         collaterals: vec![],
```

Comment:
@politeWall this field does not exist anymone now it is a single `collateral` please check all the fields and make sure theey are aligned with the native module MTP structure

Reply:
i will do that right away